web/src/beta/components/Modal/ModalFrame/index.tsx

```diff
@@ -17,7 +19,8 @@ export type Props = {
 
 const Modal: React.FC<Props> = ({ className, size, isVisible, title, onClose, children }) => {
   const ref = useRef<HTMLDivElement>(null);
-  useClickAway(ref, () => onClose?.());
+  // useClickAway(ref, () => onClose?.());
```

Comment:
This registers the click on `select` field as it sits outside the HTML element of the modal and closes the entre modal